src/coreclr/tools/dotnet-pgo/Program.cs

```diff
@@ -451,18 +452,20 @@ static int InnerMergeMain(CommandLineOptions commandLineOptions)
             }
         }
 
-        static int InnerCompareMibcMain(CommandLineOptions options)
+        private int InnerCompareMibcMain()
         {
+            var paths = Get(_command.InputFilesToCompare);
+
             // Command line parser should require exactly 2 files
-            Trace.Assert(options.CompareMibc?.Count == 2);
-            FileInfo file1 = options.CompareMibc[0];
-            FileInfo file2 = options.CompareMibc[1];
+            Trace.Assert(paths.Length == 2);
+            string file1 = paths[0];
+            string file2 = paths[1];
 
             // Look for the shortest unique names for the input files.
-            string name1 = file1.Name;
-            string name2 = file2.Name;
-            string path1 = Path.GetDirectoryName(file1.FullName);
-            string path2 = Path.GetDirectoryName(file2.FullName);
+            string name1 = paths[0];
+            string name2 = paths[1];
```

Comment:
```suggestion
            string name1 = Path.GetFileName(paths[0]);
            string name2 = Path.GetFileName(paths[1]);
```

Can you double check that this looks correct?

Reply:
In general, what is the reason to get rid of these `FileInfo`s in options? I believe System.CommandLine has good support for them (e.g. `new Option<FileInfo>("...").ExistingOnly()`).

Reply:
Yes, your suggestion is correct. Thanks!